packages/rpc-transformers/src/response-transformer-allowed-numeric-values.ts

```diff
@@ -37,3 +37,16 @@ export const jsonParsedAccountsConfigs = [
     ['data', 'parsed', 'info', 'commission'],
     ['data', 'parsed', 'info', 'votes', KEYPATH_WILDCARD, 'confirmationCount'],
 ];
+export const instructionConfig = [['accounts', KEYPATH_WILDCARD], ['programIdIndex'], ['stackHeight']];
+export const innerInstructionsConfigs = [
+    ['index'],
+    ...instructionConfig.map(c => ['instructions', KEYPATH_WILDCARD, ...c]),
+];
+export const messageConfig = [
+    ['addressTableLookups', KEYPATH_WILDCARD, 'writableIndexes', KEYPATH_WILDCARD],
+    ['addressTableLookups', KEYPATH_WILDCARD, 'readonlyIndexes', KEYPATH_WILDCARD],
+    ...instructionConfig.map(c => ['instructions', KEYPATH_WILDCARD, ...c]),
+    ['header', 'numReadonlySignedAccounts'],
+    ['header', 'numReadonlyUnsignedAccounts'],
+    ['header', 'numRequiredSignatures'],
+] as const;
```

Comment:
Did a little compressing of the configuration that prevents bigint upcasts for certain values, while also adding `stackHeight` to the list. Please go over with a fine-toothed comb in case I missed anything.